Java/FrequencyList.java

```diff
@@ -0,0 +1,40 @@
+package lfu;
+import java.util.LinkedList;
+import java.util.ListIterator;
+
+class FrequencyList<T> {
```

Comment:
Most of this java code has too many lint / formatting errors. Did you try running a java linter on it ?